cypress/e2e/renderer.feature

```diff
@@ -42,3 +42,45 @@ Feature: Test plugin renderer
     And I expect "#link-__workflow-__next-wfstep5-wfstep6" is visible
     And I expect "#link-__workflow-__next-wfstep6-wfstep7" is visible
     And I expect "#link-__workflow-__next-wfstep7-wfstep8" is visible
+
+Scenario: Reset components with a selected component that does not exist
+  Then I expect ".component" appear 16 times on screen
+  And I expect ".component-DefaultModel" appear 12 times on screen
```

Comment:
align I

Reply:
Not done, but link to a linter improvement.